gossip/src/crds_value.rs

```diff
@@ -157,9 +166,15 @@ impl CrdsData {
             3 => CrdsData::AccountsHashes(AccountsHashes::new_rand(rng, pubkey)),
             4 => CrdsData::Version(Version::new_rand(rng, pubkey)),
             5 => CrdsData::Vote(rng.gen_range(0..MAX_VOTES), Vote::new_rand(rng, pubkey)),
-            _ => CrdsData::EpochSlots(
+            6 => CrdsData::RestartLastVotedForkSlots(
                 rng.gen_range(0..MAX_EPOCH_SLOTS),
                 EpochSlots::new_rand(rng, pubkey),
+                rng.gen_range(0..512),
+                Hash::new_unique(),
+            ),
+            _ => CrdsData::EpochSlots(
+                rng.gen_range(0..MAX_RESTART_LAST_VOTED_FORK_SLOTS),
+                EpochSlots::new_rand(rng, pubkey),
```

Comment:
This looks wrong.
You are using `MAX_EPOCH_SLOTS` for `RestartLastVotedForkSlots`
and `MAX_RESTART_LAST_VOTED_FORK_SLOTS` for `EpochSlots`.

Reply:
Oops, good catch, thank for pointing that out.